### ISSUE TITLE
The Dexterity-reference from the training and the dexterity docs should both move into docs

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1298]
The Dexterity-reference from the training (https://training.plone.org/mastering-plone/dexterity_reference.html) and the dexterity docs should both move into `plone/documentation`. I would not pull the dexterity docs in (as a submodule) but rather move it there since it is very very "core".

## Sources

- https://github.com/plone/plone.app.dexterity/tree/master/docs
- https://github.com/plone/training/blob/main/docs/mastering-plone/dexterity_reference.md

## Advantages

- Core docs should be in the `plone/documentation`, instead of scattered across multiple repositories.
- More likely to get contributions through the `plone/documentation` repository. No need to explain, "Oh, sorry, you have to go over to `plone/plone.app.dexterity` to update this one section of the docs."
- No need to deal with submodules.

## Disadvantages

- Docs would not be kept together with the project.

## To Dos

- [ ] https://github.com/plone/documentation/pull/1455
- [ ] https://github.com/plone/documentation/issues/1309
- [ ] Import https://github.com/plone/training/blob/main/docs/mastering-plone/dexterity_reference.md

## See also

- https://github.com/plone/documentation/issues/1445
- https://github.com/plone/documentation/issues/1143
- https://github.com/plone/documentation/issues/707

_Originally posted by @pbauer in https://github.com/plone/documentation/pull/1290#discussion_r964473358_

**Comment by jensens** [comment]
+100 

Also consider to include the plone.behavior readme https://github.com/plone/plone.behavior/blob/master/README.rst (and reduce it to a link to the docs).

**Comment by stevepiercy** [comment]
> Also consider to include the plone.behavior readme https://github.com/plone/plone.behavior/blob/master/README.rst (and reduce it to a link to the docs).

Agreed. Created new issue: https://github.com/plone/documentation/issues/1309

Are there other external docs that should be moved into `plone/documentation`?

**Comment by stevepiercy** [comment]
## To Dos

- [ ] https://github.com/plone/documentation/pull/1455
- [ ] https://github.com/plone/documentation/issues/1309
- [ ] Import https://github.com/plone/training/blob/main/docs/mastering-plone/dexterity_reference.md
